README.md

```diff
@@ -196,19 +196,19 @@ val client = Client[PickleType, ClientResult](Transport, MyLogHandler)
 Define it when creating the `Router`:
 ```scala
 object MyLogHandler extends LogHandler[ServerResult[_]] {
-  def logRequest[T](path: List[String], argumentObject: Any, result: ServerResult[T]): ServerResult[T] = ???
+  def logRequest[T](method: Method, argumentObject: Any, result: ServerResult[T]): ServerResult[T] = ???
 }
 
 val router = Router[PickleType, ServerResult](MyLogHandler)
 ```
 
 ### Method overloading
 
-When overloading methods with different parameter lists, sloth does not have a unique path (because it is derived from the trait name and the method name). Here you will need to provide your own path name:
+When overloading methods with different parameter lists, sloth cannot have uniquely identify the method (because it is referenced with the trait name and the method name). Here you will need to provide a custom name:
```

Comment:
```suggestion
When overloading methods with different parameter lists, sloth cannot uniquely identify the method (because it is referenced with the trait name and the method name). Here you will need to provide a custom name:
```